backend/api/v1/models.py

```diff
@@ -125,7 +143,61 @@ class Skill(db.Model):
     slug = db.Column(IdentifierType, primary_key=True)
     name = db.Column(db.String(50), nullable=False)
 
-    content = db.relationship("Content", back_populates="skills", secondary="content_skill_association")
+    content = db.relationship(
+        "Content", back_populates="skills", secondary="content_skill_association"
+    )
 
-    def __repr__(self):
+    def __repr__(self):  # pragma: no cover
         return f"<Skill(name='{self.name}')>"
+
+
+class Submission(db.Model):
+    __tablename__ = "submissions"
+
+    slug = db.Column(IdentifierType, primary_key=True)
```

Comment:
Does this make sense? Shouldn't we use auto-incremented numbers here?

Reply:
As the submission slug, I use a hashed version of the content and the current timestamp.
Both is fine for me, but as we use slugs everywhere, I though it might be beneficial to stay with it. In principle, I do not care. Shall I change it?

Reply:
Let's use whatever works. I was just wondering if this `slug` needs to be (manually) created somehow. If you create it automatically and it's reproducible I'm fine with it. Thanks for the clarification.